shared/lib/transactions-controller-utils.js

```diff
@@ -35,6 +35,52 @@ export function calcTokenAmount(value, decimals) {
   return new BigNumber(String(value)).div(multiplier);
 }
 
+export function stripAddressFromLogTopic(addressLogTopic) {
+  return `0x${addressLogTopic.replace(/^0x000000000000000000000000/u, '')}`;
+}
+
+/**
+ * Extracts token transfers from a txReceipt by filtering through logs.
+ *
+ * @param txReceipt - A transaction receipt.
+ * @returns {{ contractAddress: string, to: string, from: string }} A list of token transfers from a txReceipt.
+ */
+export function getTokenTransfersFromTxReceipt(txReceipt) {
+  const txReceiptLogs = txReceipt?.logs;
+  if (!txReceiptLogs || txReceipt?.status === '0x0') {
+    return [];
+  }
+  return txReceiptLogs
+    .map((txReceiptLog) => {
+      if (!txReceiptLog.topics) {
+        return undefined;
+      }
```

Comment:
you can use a concise version and keep it on one line :
`if (!txReceiptLog.topics) return undefined`